flutter_loggy/lib/flutter_loggy/ui/widgets/loggy_stream_item_widget.dart

```diff
@@ -1,4 +1,4 @@
-part of flutter_loggy;
+part of '../../../flutter_loggy.dart';
```

Comment:
Why does this need changing in all the files?

Reply:
This does not change any behavior. The [official dart-lint](https://dart.dev/tools/linter-rules/use_string_in_part_of_directives) recommends to do it this way.